config/pitest-suppressions/pitest-imports-suppressions.xml

```diff
@@ -19,11 +19,11 @@
   </mutation>
 
   <mutation unstable="false">
-    <sourceFile>PkgImportControl.java</sourceFile>
-    <mutatedClass>com.puppycrawl.tools.checkstyle.checks.imports.PkgImportControl</mutatedClass>
-    <mutatedMethod>&lt;init&gt;</mutatedMethod>
+    <sourceFile>RedundantImportCheck.java</sourceFile>
```

Comment:
@Kevin222004 , please look at this it sounds like we found how to kill mutation

Reply:
@romani we can not remove this suppression because https://github.com/checkstyle/checkstyle/pull/13249#discussion_r1233416193 

I have tested this https://github.com/checkstyle/checkstyle/pull/13906 it is not getting killed.
And may be it will not.

@suniti0804 can you please share why you update this file and how using cli or by your own?

Reply:
I have updated this file as in the failed checks log this was being shown. I updated it using CLI.

Reply:
@romani I am not sure how this is been killed in https://github.com/checkstyle/checkstyle/commit/0524e3ef7221c102895b173c46db12e671561a82. because I have tested similar code changes but I didn't find that mutation is getting killed.

Generally, this will not happen. but we have seen this before I don't remember exactly where.